cypress.config.js

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
```

Comment:
To get around the typing issue that you're running into currently, you could try removing the `defineConfig` and do just return and:

```export default {
  video: false,
  e2e: {
    baseUrl: 'http://localhost:1337/',
    requestBaseUrl: 'http://api.localhost:1337'
  },
};```

Reply:
@Lakitna I understand that. The next thing I would recommend is to [file an issue](https://github.com/cypress-io/cypress/issues/new?assignees=&labels=&template=1-bug-report.yml) in the core repo as there may be others that have run into this issue too.

Reply:
Create https://github.com/cypress-io/cypress/issues/22127